app/lib/replay/Problems.ts

```diff
@@ -48,86 +55,241 @@ export interface BoltProblem extends BoltProblemDescription {
   solution?: BoltProblemSolution;
 }
 
-export type BoltProblemInput = Omit<BoltProblem, "problemId" | "timestamp">;
+export type BoltProblemInput = Omit<BoltProblem, 'problemId' | 'timestamp'>;
+
+// Helper functions for mapping between Supabase and Bolt types
+function mapSupabaseToBoltStatus(status: Database['public']['Tables']['problems']['Row']['status']): BoltProblemStatus {
```

Comment:
Looks like the AI had a hicc-up here:

1. The existing "Bolt `status`" is capitalized, but here we return them all lowercase.
2. The two functions are copy's of one another but shouldn't be.
3. We don't need to convert to all lowercase.

Can we remove these `map` functions?